L1Trigger/Phase2L1ParticleFlow/test/make_l1ct_binaryFiles_TDR_cfg.py

```diff
@@ -0,0 +1,153 @@
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '123X_mcRun4_realistic_v3', '')
+
+process.load('L1Trigger.Phase2L1ParticleFlow.l1ctLayer1_cff')
+process.load('L1Trigger.Phase2L1ParticleFlow.l1ctLayer2EG_cff')
+process.load('L1Trigger.L1TTrackMatch.l1tGTTInputProducer_cfi')
+process.load('L1Trigger.VertexFinder.l1tVertexProducer_cfi')
+process.l1tVertexFinderEmulator = process.l1tVertexProducer.clone()
+process.l1tVertexFinderEmulator.VertexReconstruction.Algorithm = "fastHistoEmulation"
+process.l1tVertexFinderEmulator.l1TracksInputTag = cms.InputTag("l1tGTTInputProducer", "Level1TTTracksConverted")
+from L1Trigger.Phase2L1GMT.gmt_cfi import l1tStandaloneMuons
+process.l1tSAMuonsGmt = l1tStandaloneMuons.clone()
+
+from L1Trigger.Phase2L1ParticleFlow.l1tSeedConePFJetProducer_cfi import l1tSeedConePFJetEmulatorProducer
+from L1Trigger.Phase2L1ParticleFlow.l1tDeregionizerProducer_cfi import l1tDeregionizerProducer
+from L1Trigger.Phase2L1ParticleFlow.l1tJetFileWriter_cfi import l1tSeededConeJetFileWriter
+process.l1tLayer2Deregionizer = l1tDeregionizerProducer.clone()
+process.l1tLayer2SeedConeJetsCorrected = l1tSeedConePFJetEmulatorProducer.clone(L1PFObject = cms.InputTag('l1tLayer2Deregionizer', 'Puppi'),
+                                                                                doCorrections = cms.bool(True),
+                                                                                correctorFile = cms.string("L1Trigger/Phase2L1ParticleFlow/data/jecs/jecs_20220308.root"),
+                                                                                correctorDir = cms.string('L1PuppiSC4EmuJets'))
+process.l1tLayer2SeedConeJetWriter = l1tSeededConeJetFileWriter.clone(jets = "l1tLayer2SeedConeJetsCorrected")
+
+process.l1tLayer1Barrel9 = process.l1tLayer1Barrel.clone()
+process.l1tLayer1Barrel9.puAlgo.nFinalSort = 32
+process.l1tLayer1Barrel9.regions[0].etaBoundaries = [ -1.5, -0.5, 0.5, 1.5 ] 
+process.l1tLayer1Barrel9.boards=cms.VPSet(
+        cms.PSet(
+            regions=cms.vuint32(*[0+9*ie+i for ie in range(3) for i in range(3)])),
+        cms.PSet(
+            regions=cms.vuint32(*[3+9*ie+i for ie in range(3) for i in range(3)])),
+        cms.PSet(
+            regions=cms.vuint32(*[6+9*ie+i for ie in range(3) for i in range(3)])),
+    )
+
+process.l1tLayer1BarrelTDR = process.l1tLayer1Barrel.clone()
+process.l1tLayer1BarrelTDR.regionizerAlgo = cms.string("TDR")
+process.l1tLayer1BarrelTDR.regionizerAlgoParameters = cms.PSet(
+        nTrack = cms.uint32(22),
+        nCalo = cms.uint32(15),
+        nEmCalo = cms.uint32(12),
+        nMu = cms.uint32(2),
+        nClocks = cms.int32(162),
+        doSort = cms.bool(False),
+        bigRegionEdges = cms.vint32(-560, -80, 400, -560),
+        debug = cms.untracked.bool(True)
+    )
```

Comment:
I'd prefer if you just added it to the other cfg.py instead of creating a new one to avoid risks of diverging on other things

Reply:
I agree. I changed it.